scio-examples/src/main/java/org/apache/beam/examples/common/ExampleCloudSqlOptions.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.beam.examples.common;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.Validation;
+
+public interface ExampleCloudSqlOptions extends PipelineOptions {
+
+  /**
+   * Get the username as command line option
+   */
+  @Description("Cloud Sql database username")
+  @Validation.Required
+  String getUsername();
```

Comment:
These argument keys may look confusing on the commandline, e.g. `--username=... --password=...`. Make it more explicit like `--cloudSqlUser=... --cloudSqlPass=...` ?